scripts/gulp/tasks/bump-versions.ts

```diff
@@ -1,71 +1,44 @@
 import { dest, src, task } from 'gulp';
 
-const modify = require('gulp-json-modify');
-const seq = require('gulp-sequence');
+const fs = require('fs');
+const through = require('through2');
 
 const VERSION_APPENDIX = process.env.NEBULAR_VERSION_APPENDIX;
 const VERSION = process.env.NEBULAR_VERSION || require('../../../package.json').version +
   (VERSION_APPENDIX ? '-' + VERSION_APPENDIX : '');
+const FRAMEWORK_ROOT = './src/framework';
```

Comment:
we need to exclude icons

Reply:
It's excluded by the keep [nebular packages function](https://github.com/akveo/nebular/pull/861/files#diff-a15f9f419fbd7ee4a8a509907779707aR24)